### PR TITLE
survey: Return survey results for single-page assessments too.

### PR DESCRIPTION
**Pull request by breville**

### CODE REVIEW

**Comment by ashercodeorg** [comment]
Is this meant to be `staging` or `staging-next`?

**Comment by mehalshah** [comment]
LGTM but echoing Asher's comments - this should be against staging-next

**Comment by pcardune** [comment]
should there be a corresponding change to unit tests for this?

File: dashboard/test/controllers/api_controller_test.rb [hunk 1]
```diff
@@ -360,6 +360,113 @@ def make_progress_in_section(script)
     assert_equal expected_response, JSON.parse(@response.body)
   end
 
+  test "should get surveys for section with script with single page anonymous level_group assessment" do
```
Comment (pcardune) [inline dashboard/test/controllers/api_controller_test.rb:363]: 👏

File: dashboard/test/ui/step_definitions/steps.rb [hunk 2]
```diff
@@ -767,10 +767,11 @@ def generate_teacher_student(name, teacher_authorized)
   steps %Q{
     Then I am on "http://code.org/teacher-dashboard#/sections"
     And I wait to see ".jumbotron"
+    And I wait for 3 seconds
```
Comment (breville) [inline dashboard/test/ui/step_definitions/steps.rb]: @Bjvanminnen I made a few changes that seemed to help here.

File: dashboard/test/ui/step_definitions/steps.rb [hunk 3]
```diff
@@ -767,10 +767,11 @@ def generate_teacher_student(name, teacher_authorized)
...
     And I click selector ".btn-white:contains('New section')" once I see it
```
Comment (breville) [inline dashboard/test/ui/step_definitions/steps.rb:809]: @Bjvanminnen Any idea why a 3 second delay is needed despite having `once I see it` here?

Comment (Bjvanminnen) [reply dashboard/test/ui/step_definitions/steps.rb:809]: Nope.

**Pull request merged**